### DISCUSSION TITLE
Latest Zotero Beta breaks mdnotes

### DISCUSSION THREAD

**Discussion by cjpoor** [argenos/zotero-mdnotes#132] [General]
@argenos See [this discussion](forums.zotero.org/discussion/91449/add-ons-broken-in-the-latest-beta-version-bbt-zutilo-etc) regarding changes in Zotero that require updates to plugins:
forums.zotero.org/discussion/91449/add-ons-broken-in-the-latest-beta-version-bbt-zutilo-etc

**Comment by argenos** [comment]
From that discussion, I honestly did not understand what exactly do I need to update?

**Comment by argenos** [comment]
@cjpoor please only post this on a single place to minimize notifications. I'm locking this discussion since this was posted in #130. I've also opened (and pinned) #133 for any future upstream changes that need to be noted.
